ts/src/local/conductor.ts

```diff
@@ -249,14 +249,12 @@ export class Conductor implements IConductor {
     );
     const startPromise = new Promise<void>((resolve) => {
       runConductorProcess.stdout.on("data", (data: Buffer) => {
-        logger.debug(`starting conductor\n${data}`);
```

Comment:
we still need a line like this, piping the info through. WASM_LOG debug statements are outputted here, instead of `stderr` for some reason lolol

Reply:
d'oh, of course, just without the "starting conductor". Sorry, I'll bring that back.